### PR TITLE
Lavaland elites rules are now shown on victory

### PR DESCRIPTION
**Pull request by Rayforx**
## What Does This PR Do

Gives the rules for elites when they win, so if someone has not checked the rules, they are able to read them more easily.
## Why It's Good For The Game

It can be pretty hard to find the rules when they are hidden in the antag policy, so this helps them be more readily available to people.
## Images of changes

image

## Testing

Spawned a tumor, added the hive
Then died, and saw the message
## Changelog
:cl:
tweak: Lavaland Elites rules will be shown on victory
/:cl:

### CODE REVIEW

**Review by Burzah — APPROVED** [review]

**Review by DGamerL — COMMENTED** [review]
Could you remake this into something of the style of
```js
var/list/text_to_send = list()
text_to_send += "This is the first to_chat() string"
text_to_send += "This is the second to_chat() string"
to_chat(mychild, text_to_send.Join(""))
```
Feel free to ask for help on this!

**Comment by Rayforx** [comment]
> Could you remake this into something of the style of
> 
> ```js
> var/list/text_to_send = list()
> text_to_send += "This is the first to_chat() string"
> text_to_send += "This is the second to_chat() string"
> to_chat(mychild, text_to_send.Join(""))
> ```
> 
> Feel free to ask for help on this!

I will try and get to it either later today or tomorrow.

**Comment by DGamerL** [comment]
image

The newlines and missing spaces have been fixed in the latest commit

**Review by DGamerL — APPROVED** [review]
Very objective review 👍

File: code/modules/mob/living/simple_animal/hostile/mining/elites/elite.dm [hunk 1]
```diff
@@ -398,14 +398,21 @@ While using this makes the system rely on OnFire, it still gives options for tim
...
+		text += "<b>Your max health has been halved, but can now heal by standing on your tumor. Note, it's your only way to heal. </b>"
+		text += "<b>Bear in mind, if anyone interacts with your tumor, you'll be resummoned here to carry out another fight. In such a case, you will regain your full max health. </b>"
+		text += "<b>Also, be wary of your fellow inhabitants, they likely won't be happy to see you! \n</b>"
+		text += "<span class='big bold'>Note that you are a lavaland monster, and thus not allied to the station. </span>"
+		text += "<span class='big bold'You should not cooperate or act friendly with any station crew unless under extreme circumstances! </span>"
+		text += "<span class='warning'>Do not attack the Mining Station or Labour Camp, unless the Shaft Miner you are actively fighting runs into the Station/Camp. </span"
+		text += "<span class='warning'>After they are killed, you must withdraw. If you wish to continue attacking the Station, you MUST ahelp. </span>"
+		text += "<span class='warning'>If teleported to the Station by jaunter, you are allowed to attack people on Station, until you get killed.</span>"
```
Comment (Burzah) [inline code/modules/mob/living/simple_animal/hostile/mining/elites/elite.dm:413]:
```suggestion
		text += "<span class='warning'>As the life in the activator's eyes fade, the forcefield around you dies out and you feel your power subside.</span>"
		text += "<span class='warning'>Despite this inferno being your home, you feel as if you aren't welcome here anymore.</span>"
		text += "<span class='warning'>Without any guidance, your purpose is now for you to decide.\n</span>"
		text += "<b>Your max health has been halved, but can now heal by standing on your tumor. Note, it's your only way to heal.</b>"
		text += "<b>Bear in mind, if anyone interacts with your tumor, you'll be resummoned here to carry out another fight. In such a case, you will regain your full max health.</b>"
		text += "<b>Also, be wary of your fellow inhabitants, they likely won't be happy to see you! \n</b>"
		text += "<span class='big bold'>Note that you are a lavaland monster, and thus not allied to the station.</span>"
		text += "<span class='big bold>'You should not cooperate or act friendly with any station crew unless under extreme circumstances!</span>"
		text += "<span class='warning'>Do not attack the Mining Station or Labour Camp, unless the Shaft Miner you are actively fighting runs into the Station/Camp.</span>"
		text += "<span class='warning'>After they are killed, you must withdraw. If you wish to continue attacking the Station, you MUST ahelp.</span>"
		text += "<span class='warning'>If teleported to the Station by jaunter, you are allowed to attack people on Station, until you get killed.</span>"
```

File: code/modules/mob/living/simple_animal/hostile/mining/elites/elite.dm [hunk 2]
```diff
@@ -398,14 +398,21 @@ While using this makes the system rely on OnFire, it still gives options for tim
...
+		text += "<b>Your max health has been halved, but can now heal by standing on your tumor. Note, it's your only way to heal.</b>"
+		text += "<b>Bear in mind, if anyone interacts with your tumor, you'll be resummoned here to carry out another fight. In such a case, you will regain your full max health.</b>"
+		text += "<b>Also, be wary of your fellow inhabitants, they likely won't be happy to see you! \n</b>"
+		text += "<span class='big bold'>Note that you are a lavaland monster, and thus not allied to the station.</span>"
+		text += "<span class='big bold>'You should not cooperate or act friendly with any station crew unless under extreme circumstances!</span>"
+		text += "<span class='warning'>Do not attack the Mining Station or Labour Camp, unless the Shaft Miner you are actively fighting runs into the Station/Camp.</span>"
+		text += "<span class='warning'>After they are killed, you must withdraw. If you wish to continue attacking the Station, you MUST ahelp.</span>"
+		text += "<span class='warning'>If teleported to the Station by jaunter, you are allowed to attack people on Station, until you get killed.</span>"
+		to_chat(mychild, text.Join(""))
```
Comment (DGamerL) [inline code/modules/mob/living/simple_animal/hostile/mining/elites/elite.dm:414]:
```suggestion
		to_chat(mychild, text.Join(" "))
```

**Review by Burzah — APPROVED** [review]

**Pull request merged**